pkgs/development/python-modules/py-radix/default.nix

```
@@ -0,0 +1,28 @@
+{ stdenv
+, buildPythonPackage
+, fetchFromGitHub
+, coverage
+, nose
+}:
+
+buildPythonPackage rec {
+  pname = "py-radix";
+  version = "v0.10.0";
```

```suggestion
  version = "0.10.0";
```
Sorry I missed this before.
Version number must start with a digit, otherwise Nix will see the package as being named `py-radix-v`